docs/interactions/structured-messages/select.md

```diff
@@ -220,3 +220,78 @@ Primary parameters are used by default, however, some parameters are unique or o
 |-|-|
 | **`structured_content.items.title`** | Truncated to 25 characters. |
 | **`structured_content.items.payload`** | Automatically gets populated as a random hex if blank. |
+
+## Example: WhatsApp (List Messages)
+
+The following example uses WhatsApp with list messages.
+
+<img class="img-fluid" width="398" src="../../../img/structured-messages-select-whatsapp-1.png">
+<img class="img-fluid" width="398" src="../../../img/structured-messages-select-whatsapp-2.png">
+
+### JSON Body
+
+```json
+{
+  "source_id": "<source_id>",
+  "in_reply_to_id": "<in_reply_to_id>",
+  "body": "What do you wish?",
+  "structured_content": {
+    "type": "select",
+    "button": "Choose an option",
```

Comment:
Can you add all the specific attributes in this example payload? Ideally the screenshots should be the result of this payload.

Reply:
fixed